### ISSUE TITLE
Xiaomi rice cook mode stopped working from version 0.7.25 and above

### ISSUE THREAD

**Issue by niruse** [al-one/hass-xiaomi-miot#2068]
https://home.miot-spec.com/s/chunmi.cooker.wy3

Added image
![image](https://github.com/user-attachments/assets/ba79ef6c-cd92-45a1-8da3-e62b0845949e)
Everything working properly in version 0.7.24.
Added image from this version
![image](https://github.com/user-attachments/assets/4a9caa44-46a4-48f4-823c-64d589a261e1)

**Comment by al-one** [comment]
Try master branch.
```bash
wget -O - https://get.hacs.vip | DOMAIN=xiaomi_miot ARCHIVE_TAG=master bash -
```

**Comment by al-one** [comment]
Fixed in v1.0.5.

**Comment by niruse** [comment]
Problem solved:-)
